### DISCUSSION TITLE
where to find gitkey.dart?

### DISCUSSION THREAD

**Discussion by Sahal56** [Hash-Studios/Prism#252] [Show and tell]
import 'package:Prism/gitkey.dart';

this generates error.
Can anyone help me find ?
i Already fetch packages though im gettin this.!!
<img width="1440" alt="Screenshot 2022-12-15 at 1 33 57 PM" src="https://user-images.githubusercontent.com/63746256/207805562-a3802788-de13-484b-a12c-af7497ff2afe.png">

**Comment by codenameakshay** [comment]
this has been fixed, not needed anymore
